tests/vm_util_test.py

```diff
@@ -17,10 +17,56 @@
 import unittest
 
 import mock
+import os
 
 from perfkitbenchmarker import vm_util
 
 
+if os.name is 'posix':
```

Comment:
Sorry to be slow responding to this - I was traveling last week and missed this update.

I agree that this is overkill since there's no new logic.  You could mock `subprocess.Popen` itself (and verify that it was constructed properly). Otherwise removing the test is fine.